main/iptables/APKBUILD

```diff
@@ -21,16 +21,11 @@ source="https://www.netfilter.org/projects/iptables/files/iptables-$pkgver.tar.b
 builddir="$srcdir/$pkgname-$pkgver"
 
 prepare() {
-	default_prepare
```

Comment:
Why would you remove `default_prepare`??

Reply:
fixed